js/cryptocom.js

```diff
@@ -0,0 +1,1037 @@
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOpenOrders requires a `symbol` argument');
+        }
+        const market = this.market (symbol);
+        const request = {
+            'instrument_name': market['id'],
+        };
+        const response = await this.privatePostPrivateGetOpenOrders (this.extend (request, params));
+        // {
+        //     "id": 11,
+        //     "method": "private/get-open-orders",
+        //     "code": 0,
+        //     "result": {
+        //       "count": 1177,
+        //       "order_list": [
+        //         {
+        //           "status": "ACTIVE",
+        //           "side": "BUY",
+        //           "price": 1,
+        //           "quantity": 1,
+        //           "order_id": "366543374673423753",
+        //           "client_oid": "my_order_0002",
+        //           "create_time": 1588760643829,
+        //           "update_time": 1588760644292,
+        //           "type": "LIMIT",
+        //           "instrument_name": "ETH_CRO",
+        //           "cumulative_quantity": 0,
+        //           "cumulative_value": 0,
+        //           "avg_price": 0,
+        //           "fee_currency": "CRO",
+        //           "time_in_force": "GOOD_TILL_CANCEL"
+        //         },
+        //         {
+        //           "status": "ACTIVE",
+        //           "side": "BUY",
+        //           "price": 1,
+        //           "quantity": 1,
+        //           "order_id": "366455245775097673",
+        //           "client_oid": "my_order_0002",
+        //           "create_time": 1588758017375,
+        //           "update_time": 1588758017411,
+        //           "type": "LIMIT",
+        //           "instrument_name": "ETH_CRO",
+        //           "cumulative_quantity": 0,
+        //           "cumulative_value": 0,
+        //           "avg_price": 0,
+        //           "fee_currency": "CRO",
+        //           "time_in_force": "GOOD_TILL_CANCEL"
+        //         }
+        //       ]
+        //     }
+        // }
+        const data = this.safeValue (response, 'result', {});
+        const resultList = this.safeValue (data, 'order_list', []);
+        let orders = this.parseOrders (resultList, market, since, limit);
+        orders = this.filterBy (orders, 'symbol', symbol);
+        return orders;
+    }
+
+    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let orders = await this.fetchOrders (symbol, since, limit, params);
+        orders = this.filterByArray (orders, 'status', [ 'closed', 'canceled' ], false);
+        return orders;
+    }
+
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchMyTrades requires a `symbol` argument');
+        }
```

Comment:
But why? In https://exchange-docs.crypto.com/spot/index.html?javascript#private-get-trades it says that you can omit the instrument_name and get all trades.